benchmark/fs/bench-glob.js

```diff
@@ -25,10 +25,26 @@ async function main(config) {
   bench.start();
 
   for (let i = 0; i < config.n; i++) {
-    if (mode === 'async') {
-      noDead = await fs.promises.glob(pattern, { cwd: fullPath, recursive });
-    } else {
-      noDead = fs.globSync(pattern, { cwd: fullPath, recursive });
+    switch (mode) {
+      case 'sync':
+        noDead = fs.globSync(pattern, { cwd: fullPath, recursive });
```

Comment:
I think we should build the options object outside the loop

Reply:
Sure, implemented in https://github.com/nodejs/node/pull/58417/commits/531d6be411c8f79d247d3c6ecd2aa9ae718fb582